### PR TITLE
Updated Database.md

### PR DESCRIPTION
**Pull request by apu52**
Updated Datadog and stripe student offers

# Issue Number : 
.
.
.
# Modified File Path : 
.
.
.
# Description : 
.
.
.
# Mentions : 
@ShreyamMaity

### CODE REVIEW

File: Database/database.md [hunk 1]
```diff
@@ -76,7 +76,7 @@
 |[DNS simple](https://dnsimple.com/signup)|DNS Simple student license|DNS data observation|
 |[Orchestrate.io](https://)|Free developer account|database service|
 |[Send grid](https://)|Student License|STMP mail system|
-|[Bitnami](https://bitnami.com/)|Student License free|pre-packaged images for automated setup of popular server software on Windows, Mac OS|
```
Comment (ShreyamMaity) [inline Database/database.md:79]: bitnami got removed in this pr . please check.

**Review by ShreyamMaity — CHANGES_REQUESTED** [review]

**Comment by ShreyamMaity** [comment]
@apu52 check the bitnami link

**Review by apu52 — COMMENTED** [review]
solved

**Comment by ShreyamMaity** [comment]
> solved
> 

If you open database.md of this pull request you can find that bitnami link got deleted. I think when you've created this pull request, your fork was not updated with the recent changes that's why.

**Pull request closed without merging**